genie-agent/src/main/java/com/netflix/genie/agent/cli/ArgumentDelegates.java

```diff
@@ -98,4 +99,12 @@
         boolean isJobRequestedViaAPI();
     }
 
+    /**
+     * Delegate for job folder cleanup options.
+     */
+    interface CleanupArguments {
+
+        CleanupStrategy getCleanupStrategy();
```

Comment:
Maybe because this is package protected it's not asking for javadoc here?

Reply:
I've noticed for sure that javadoc is not required on package protected, only public.
However i thought interfaces were default public, so i would have expected an error here.

(Commenting on this interface i added is in line with the rest of the file)